src/sqlfluff/core/templaters/base.py

```diff
@@ -180,39 +177,6 @@ def _find_slice_indices_of_templated_pos(
             raise ValueError("Position Not Found")
         return first_idx, last_idx
 
-    @cached_property
-    def raw_slice_block_info(self) -> RawSliceBlockInfo:
```

Comment:
This function was only used by the code I removed from `discard_unsafe_fixes()`.